api/src/main/java/jakarta/data/page/KeysetAwareSlice.java

```diff
@@ -145,6 +145,14 @@ public interface KeysetAwareSlice<T> extends Slice<T> {
      */
     PageRequest.Cursor getKeysetCursor(int index);
 
+    /**
+     * Returns {@code true} when it is possible to navigate to a previous
+     * page of results.
```

Comment:
I know it would be an unusual usage pattern,  but it is technically possible to start from a cursor somewhere in the middle and navigate to previous pages from that, in which case, an implementation might not know whether or not there actually is a previous page.

```suggestion
     * Returns {@code true} when it is possible to navigate to a previous
     * page of results or if it is necessary to request a previous page in order to
     * determine whether there are more previous results.
```